src/hearings/containers/request-hearing/hearing-edit-summary/judge-details-section/judge-details-section.component.spec.ts

```diff
@@ -124,6 +124,14 @@ describe('JudgeDetailsSectionComponent', () => {
     expect(component).toBeTruthy();
   });
 
+  it('should call ngOninit', () => {
+    component.ngOnInit();
+    expect(component.showAmmendedForNeedJudge).toEqual(false);
```

Comment:
@DavidJayakumar - same spelling error here (double "m")... looks like this change is new because I didn't see it before